packages/editor-ui/src/components/WorkflowHistory/WorkflowHistoryList.vue

```diff
@@ -0,0 +1,148 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import type { UserAction } from 'n8n-design-system';
+import { useI18n } from '@/composables';
+import type { WorkflowHistory, WorkflowHistoryActionTypes } from '@/types/workflowHistory';
+import WorkflowHistoryListItem from '@/components/WorkflowHistory/WorkflowHistoryListItem.vue';
+import type { TupleToUnion } from '@/utils/typeHelpers';
+
+const props = withDefaults(
+	defineProps<{
+		items: WorkflowHistory[];
+		activeItem: WorkflowHistory | null;
+		actionTypes: WorkflowHistoryActionTypes;
+		takeItemsAtOnce: number;
```

Comment:
Does `atOnce` here mean "immediately" or "simultaneously"? If the former, would `takeItemsOnLoad` be clearer?

Reply:
Actually neither is good, using just `take` could confuse others by thinking it takes only that amount from `items`. I'll rename it to something that suggest auto load more items by the number specified